ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1889,150 +1889,150 @@ def _distributed(self) -> bool:
         _ = super().csyiqr(ncsy, key, pname='__tmpvar__', **kwargs)
-        value = self.scalar_param(self, '__tmpvar__')
+        value = self.scalar_param('__tmpvar__')
         self._run('__tmpvar__=') # Deleting temp parameter
         return value
 
     @wraps(_MapdlCore.etyiqr)
     def etyiqr(self, itype, key, **kwargs):
         """Wrap the ``etyiqr`` method to take advantage of the gRPC methods."""
         _ = super().etyiqr(itype, key, pname='__tmpvar__', **kwargs)
-        value = self.scalar_param(self, '__tmpvar__')
+        value = self.scalar_param('__tmpvar__')
         self._run('__tmpvar__=') # Deleting temp parameter
         return value
 
     @wraps(_MapdlCore.foriqr)
     def foriqr(self, node, key, **kwargs):
         """Wrap the ``foriqr`` method to take advantage of the gRPC methods."""
         _ = super().foriqr(node, key, pname='__tmpvar__', **kwargs)
-        value = self.scalar_param(self, '__tmpvar__')
+        value = self.scalar_param('__tmpvar__')
         self._run('__tmpvar__=') # Deleting temp parameter
         return value
 
     @wraps(_MapdlCore.sectinqr)
     def sectinqr(self, nsect, key, **kwargs):
         """Wrap the ``sectinqr`` method to take advantage of the gRPC methods."""
         _ = super().sectinqr(nsect, key, pname='__tmpvar__', **kwargs)
-        value = self.scalar_param(self, '__tmpvar__')
+        value = self.scalar_param('__tmpvar__')
         self._run('__tmpvar__=') # Deleting temp parameter
         return value
 
     @wraps(_MapdlCore.mpinqr)
     def mpinqr(self, mat,  iprop,  key, **kwargs):
         """Wrap the ``mpinqr`` method to take advantage of the gRPC methods."""
         _ = super().mpinqr(mat,  iprop,  key, pname='__tmpvar__', **kwargs)
-        value = self.scalar_param(self, '__tmpvar__')
+        value = self.scalar_param('__tmpvar__')
         self._run('__tmpvar__=') # Deleting temp parameter
         return value
 
     @wraps(_MapdlCore.dget)
     def dget(self, node,  idf,  kcmplx, **kwargs):
         """Wrap the ``dget`` method to take advantage of the gRPC methods."""
         _ = super().dget(node,  idf,  kcmplx, pname='__tmpvar__', **kwargs)
-        value = self.scalar_param(self, '__tmpvar__')
+        value = self.scalar_param('__tmpvar__')
         self._run('__tmpvar__=') # Deleting temp parameter
         return value
 
     @wraps(_MapdlCore.fget)
     def fget(self, node,  idf,  kcmplx, **kwargs):
         """Wrap the ``fget`` method to take advantage of the gRPC methods."""
         _ = super().fget(node,  idf,  kcmplx, pname='__tmpvar__', **kwargs)
-        value = self.scalar_param(self, '__tmpvar__')
+        value = self.scalar_param('__tmpvar__')
         self._run('__tmpvar__=') # Deleting temp parameter
         return value
 
     @wraps(_MapdlCore.erinqr)
     def erinqr(self, key, **kwargs):
         """Wrap the ``erinqr`` method to take advantage of the gRPC methods."""
         _ = super().erinqr(key, pname='__tmpvar__', **kwargs)
```

Comment:
We can suppress the output to improve stability and reduce the call time.

Also, to reduce the possibility of error, reuse the temporary variable name `"__tmpvar__"`

```suggestion
        pname = '__tmpvar__'
        super().erinqr(key, pname=pname, mute=True, **kwargs)
```

Reply:
Good suggestion. Thank you